mobile/src/slices/users/users.slice.ts

```diff
@@ -1,39 +1,11 @@
 import { createSlice } from '@reduxjs/toolkit';
 
-import { DataStatus } from '#libs/enums/enums';
-import { type ValueOf } from '#libs/types/types';
-import { type UserGetAllItemResponseDto } from '#packages/users/users';
-
-import { loadAll } from './actions';
-
-type State = {
-  users: UserGetAllItemResponseDto[];
-  dataStatus: ValueOf<typeof DataStatus>;
-};
-
-const initialState: State = {
-  users: [],
-  dataStatus: DataStatus.IDLE,
-};
+const initialState = {};
 
 const { reducer, actions, name } = createSlice({
```

Comment:
I'm not sure, but should we tottally delete `user.slice`, `user-api` files? I did exactly that on the web, but don't know if it's needed in mobile. Also there're some redundant dtos and other files in `shared` folder)

Reply:
yes, you are right